libs/design-system/src/molecules/alerts/alerts.tsx

```diff
@@ -0,0 +1,74 @@
+import { Icon } from '../..';
+import { IconType } from '../../atoms/icons/svg';
+import { cn } from '../../shared/classnames';
+
+type AlertsProps = {
+  /**
+   * The type of alert to display
+   * @default 'info'
+   * But can be one of 'success' | 'info' | 'warning' | 'error'
+   * Changing the type of alerts merely changes the color of the alert
+   */
+  type: 'success' | 'info' | 'warning' | 'error';
+
+  /**
+   * The message to display in the alert
+   */
+  message: string;
+
+  /**
+   * A link if we want the alert to be clickable
+   */
+  link?: string;
+
+  /**
+   * Some additional class names
+   */
+  className?: string;
+};
+
+const colorMapping = {
+  success: 'plasmo-bg-success plasmo-text-white-transparent90',
+  info: 'plasmo-bg-info plasmo-text-white-transparent90',
+  warning: 'plasmo-bg-pink plasmo-text-white-transparent90',
+  error: 'plasmo-bg-error plasmo-text-white-transparent90',
+};
+
+const iconsMapping = {
+  success: 'CheckCircle' as IconType,
+  info: 'InformationCircle' as IconType,
+  warning: 'ExclamationCircle' as IconType,
+  error: 'ExclamationCircle' as IconType,
+};
+
+const Alert = ({ type = 'info', className, link, message }: AlertsProps) => {
+  return link ? (
+    <a
+      href={link}
+      target="_blank"
+      rel="noreferrer"
+      className={cn(
+        'plasmo-antialiased plasmo-w-80 plasmo-h-20 plasmo-p-3 plasmo-rounded-md plasmo-flex plasmo-flex-row plasmo-items-center',
+        colorMapping[type],
+        className
+      )}>
+      <Icon type={iconsMapping[type]} className="plasmo-m-3" />
+      <p className="plasmo-m-3 plasmo-pt-[1px]">{message}</p>
+    </a>
+  ) : (
+    <span
+      className={cn(
+        'plasmo-antialiased plasmo-w-80 plasmo-h-20 plasmo-p-3 plasmo-rounded-md plasmo-flex plasmo-flex-row plasmo-items-center',
+        colorMapping[type],
+        className
+      )}>
+      <Icon type={iconsMapping[type]} className="plasmo-m-3" />
+      <p className="plasmo-m-3 plasmo-pt-[1px]">{message}</p>
+    </span>
+  );
+};
```

Comment:
```suggestion
const colorMapping = {
  success: 'plasmo-bg-success plasmo-text-white-transparent90',
  info: 'plasmo-bg-info plasmo-text-white-transparent90',
  warning: 'plasmo-bg-pink plasmo-text-white-transparent90',
  error: 'plasmo-bg-error plasmo-text-white-transparent90',
};

const iconsMapping: Record<string, IconType> = {
  success: 'CheckCircle',
  info: 'InformationCircle',
  warning: 'ExclamationCircle',
  error: 'ExclamationCircle',
};

const Alert = ({ type = 'info', className, link, message }: AlertsProps) => {
  const alert = (
    <div
      className={cn(
        'plasmo-antialiased plasmo-space-x-3 plasmo-font-semibold plasmo-w-full plasmo-p-6 plasmo-leading-4 plasmo-rounded-md plasmo-flex plasmo-flex-row plasmo-items-center',
        colorMapping[type],
        className
      )}>
      <Icon type={iconsMapping[type]} />
      <p className="plasmo-flex-wrap">{message}</p>
    </div>
  );
  return link ? (
    <a href={link} target="_blank" rel="noreferrer">
      {alert}
    </a>
  ) : (
    alert
  );
};
```

big suggestion to facilitate the process but here's some explanations:
1. I don't understand why you cast IconType. Don't hesitate to use Record (ie. Record<string, IconType>)
2. Try not to not duplicate code 👉 it's easier to wrap the main element into an anchor rather than typing it twice
3. Try not to pass fixed width or height to components, we'll handle the size when using the component. And that's why your icon was in struggle. Just don't forget the wrap in this case
4. Don't forget the leading of the text 😉

Reply:
5. Use space to separate elements rather than margins

Reply:
1. Didn't know `Record`! Nice one, thank you!
2. You're right, my bad
3. Gotcha. A lot of issues could have been avoided had I used the `space` class instead of margins. Once again: my bad.
4. Duly noted.
5. Yeah of course, my bad

Thanks!